storage/innobase/include/dict0dict.h

```diff
@@ -360,7 +360,19 @@ dict_table_autoinc_unlock(
 /*======================*/
 	dict_table_t*	table);	/*!< in/out: table */
 
+/**********************************************************************//**
+Serializes access to indexes while analyzing. */
+void
+dict_table_analyze_index_lock(
+/*====================*/
+	dict_table_t*	table);	/*!< in/out: table */
+
+void
+dict_table_analyze_index_unlock(
+/*======================*/
+	dict_table_t*	table);	/*!< in/out: table */
```

Comment:
remove /*==== */ decoration
Use doxygen style
Use param[in,out]
Remove "in/out" comment

Reply:
done